### ISSUE TITLE
Should be possible not to check complete workspace

### ISSUE THREAD

**Issue by rassie** [emacs-lsp/lsp-ui#280]
On a TypeScript/JavaScript project, `lsp-ui` initiates a Flycheck check for every open buffer. In my case, every buffer is checked with ESLint, which in turn spawns a Node process for every check for every larger typing pause. With a dozen open files, my machine is hogged down on every save of keypress with a dozen of Node processes. While I understand that this could be beneficial, especially with more efficient language servers, there should be an option to disable this behaviour, i.e. to only check the currently open buffer.

**Comment by yyoncho** [comment]
(I am no js expert/user)

Is ESlint something that is reported through by the language server or it is additional checker that you are running among lsp?

**Comment by rassie** [comment]
Yes, ESLint is an additional checker, which I added using a custom version of this [SO answer](https://emacs.stackexchange.com/questions/32900/how-to-use-web-mode-engine-specific-checkers-in-flycheck). Flycheck shows javascript-eslint as "next checkers" after `lsp-ui`.

**Comment by yyoncho** [comment]
Take a look at lsp-ui-flycheck-live-reporting docs and set it to nil. I think that it might workaround your fix.

**Comment by rassie** [comment]
It really seems to help, only one file is checked at a time. Thanks!
